pkg/kubectl/cmd/run.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const (
 	run_long = `Create and run a particular image, possibly replicated.
-Creates a replication controller to manage the created container(s).`
+Creates a deployment or job to manage the created container(s).`
```

Comment:
The problem here is that we change the defaults. Someone previously running `kubectl run` w/o specifying a generator would get a rc running their image, now they will get a deployment. I am also in favor of switching to deployments but I am not sure of changing defaults.